wal/wal_test.go

```diff
@@ -712,7 +717,12 @@ func TestOpenOnTornWrite(t *testing.T) {
 	}
 	defer os.RemoveAll(p)
 	w, err := Create(p, nil)
-	defer w.Close()
+	defer func() {
```

Comment:
`err != os.ErrInvalid` is needed in case `w.Close` gets run at line 741 https://github.com/coreos/etcd/pull/7347/commits/d901990a080f63a82d098d7c2f98b9ad13af1acd#diff-3d9bfc1d2bc5d338cfcb8baa0f9a4c6eR741
?